releases/R2024b/README.md

```diff
@@ -0,0 +1,286 @@
+   | Port and IP Address of Existing Network License Manager | Optional parameter: Specify the port number and private DNS name or private IP address of the network license manager that has already been deployed to the existing VPC. Specify it in the format port@privateDNSname, for example, `27000@ip-172-30-1-89.ec2.internal` or `27000@172.30.1.89`. By default, the license manager uses port 27000. Leave this field blank if you are deploying a new network license manager.  |
+   | Security Group of Existing Network License Manager | Optional parameter: Specify the security group of the network license manager that has already been deployed to the existing VPC. Leave this field blank if you are deploying a new network license manager. |
+
+You will also need to open the following ports in your VPC:
+
+| Port | Description |
+|------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `443` | Required for communicating with MATLAB Web App Server apps home page. |
+| `8000`, `9988` | Required for communication between MATLAB Web App Server controllers  and AWS services. These ports do not need to be open to the internet. |
+| `27000` | Required for communication between the network license manager and MATLAB Web App Server. |
+| `3389`, `22` | Required for Remote Desktop and Secure Connection functionality. This can be used for troubleshooting and debugging MATLAB Web App Server. |
+
+### How to use an existing network license manager in an existing VPC?
+If you want to use an exisiting network license manager:
+- Choose `No` for the *Deploy Network License Manager* step of the deployment.
+- Specify the IP address of the existing network license manager in the `IP Address of Existing Network License Manager` step of the deployment. You can find the private IP address in the *Outputs* tab of the existing network license manager deployment. 
+
+To use an existing network license manager, you must add the security group of the server VMs to the security group of the license manager.
+1. In the AWS management console, select the stack where the network license manager is deployed.
+1. In the *Stack details* for your stack, click **Resources**.
+1. Look for the **Logical ID** named ```SecurityGroup``` and click the corresponding URL listed under **Physical ID**. This will take you to the security group details.
+1. Click the **Inbound Rules** tab, then click **Edit Inbound Rules**.
+1. Click **Add Rule**.
+1. In the **Type** dropdown, select ```All TCP```.
+1. In the **Source**, search and add the ```matlab-webapp-server-sg``` security group. 
+1. Click **Save Rules**.
+ 
+
+## What versions of MATLAB Runtime are supported?
```

Comment:
I don't think you need this section either as his information is available in the top-level Readme.